modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -183,9 +183,6 @@ conf = {{{packageName}}}.Configuration(
         self.password = password
         """Password for HTTP basic authentication
         """
-        self.access_token = access_token
-        """Access token
-        """
```

Comment:
for oauth authentication scheme, we still need `access_token`. what about removing line 203-205 instead?

Reply:
Please PM me via Slack so that I can help you with the test.